Mlem/Views/Tabs/Feeds/Components/PostFeedView+Logic.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  PostFeedView+Logic.swift
+//  Mlem
+//
+//  Created by Eric Andrews on 2024-01-21.
+//
+
+import Dependencies
+import SwiftUI
+
+extension PostFeedView {
+    func setDefaultSortMode() async {
```

Comment:
The default-sort-mode-picking logic appears to be broken - Mlem incorrectly uses the default sort mode rather than the fallback on <0.19 instances.

Reply:
That *should* be fixed now

Reply:
Nice, looks like it's working now 👍

Reply:
Nice, looks like it's working now 👍